### PR TITLE
Api for team registration and user dashboard.

### PR DESCRIPTION
**Pull request by gaurKrishna**

### CODE REVIEW

File: accounts/serializers.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
-from rest_framework import serializers
+from django.db.models import fields
```
Comment (theyashshahs) [inline accounts/serializers.py]: why is this required?

File: accounts/models.py [hunk 2]
```diff
@@ -6,6 +6,8 @@ class Account(models.Model):
     first_name = models.CharField(max_length=50, blank=False)
     last_name = models.CharField(max_length=50, blank=True)
     mobile_number = models.CharField(max_length=12, blank=True)
+    profile = models.URLField(blank=True)
```
Comment (theyashshahs) [inline accounts/models.py]: rename it to `profile_pic` or `image_url`

File: accounts/urls.py [hunk 3]
```diff
@@ -1,4 +1,5 @@
 from django.urls import path
+from django.views.generic import base
```
Comment (theyashshahs) [inline accounts/urls.py]: remove this import

**Review by theyashshahs — COMMENTED** [review]
@gaurKrishna I have left comments, PTAL 🙂

File: accounts/serializers.py (hunk 1 above)
Comment (gaurKrishna) [reply accounts/serializers.py]: Yes, this import was not required. Removed it in the recent push.

File: accounts/models.py (hunk 2 above)
Comment (gaurKrishna) [reply accounts/models.py]: profile field has been renamed to profile_pic in account model changes are reflected in the recent push.

File: api/urls.py [hunk 4]
```diff
@@ -22,6 +22,9 @@
     path("", views.landing_page, name="landing"),
     path("admin/", admin.site.urls),
     path("account/", include("accounts.urls")),
+    path('registration/', include('registration.urls')),
+    path('events/', include('events.urls')),
+    path('team/', include('team.urls')),
```
Comment (theyashshahs) [inline api/urls.py]: why are these two routes required @gaurKrishna

**Review by theyashshahs — CHANGES_REQUESTED** [review]
@gaurKrishna remove the `url` routes for event and team, as we are already calling them in the `api/urls.py`

File: api/urls.py (hunk 4 above)
Comment (gaurKrishna) [reply api/urls.py]: Yes those two routes are not required as they have been included through simple router. 
Removing those routes in next commit.

**Review by theyashshahs — APPROVED** [review]
